src/megaapi_impl.cpp

```diff
@@ -13834,6 +13834,10 @@ void MegaApiImpl::request_error(error e)
 
 void MegaApiImpl::request_response_progress(m_off_t currentProgress, m_off_t totalProgress)
 {
+    if (!client->isFetchingNodesPendingCS())
```

Comment:
Maybe we can simply use here `MegaClient::fetchingnodes` directly??

Reply:
> since the flag might be true while the response to other commands is processed
unfortunately, it is true indeed

Reply:
> MegaClient::exec() has access to the list of commands whose HTTP request's response is being processed.
I might be wrong, but I understand that for that we'd need to access RequestsDispatcher::inflightreq. That's why I saved the flag in pendingcs. Controling the flag before or inside `request_response_progess` won't make much difference I believe. (besides megacli no longer receiving it for other comands)